addons/mail/static/src/components/message_reaction_group/message_reaction_group.xml

```diff
@@ -2,13 +2,19 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.MessageReactionGroup" owl="1">
-        <div class="o_MessageReactionGroup d-flex" t-att-class="{'o-hasUserReacted': messageReactionGroup and messageReactionGroup.hasUserReacted}" t-attf-class="{{ className }}" t-att-title="messageReactionGroup and messageReactionGroup.summary" t-on-click="messageReactionGroup and messageReactionGroup.onClick" t-ref="root">
-            <t t-if="messageReactionGroup">
+        <div class="o_MessageReactionGroup d-flex"
+             t-att-class="{'o-hasUserReacted': messageReactionGroupView and messageReactionGroupView.messageReactionGroup.hasUserReacted}"
+             t-att-title="messageReactionGroupView and messageReactionGroupView.messageReactionGroup.summary"
+             t-on-click="messageReactionGroupView and messageReactionGroupView.onClick"
+             t-on-contextmenu="messageReactionGroupView and messageReactionGroupView.onContextMenu"
+             t-attf-class="{{ className }}"
+             t-ref="root">
+            <t t-if="messageReactionGroupView">
                 <span class="o_MessageReactionGroup_content mx-1">
-                    <t t-esc="messageReactionGroup.content"/>
+                    <t t-esc="messageReactionGroupView and messageReactionGroupView.messageReactionGroup.content"/>
                 </span>
-                <span class="o_MessageReactionGroup_count mx-1" t-att-class="{'o-hasUserReacted': messageReactionGroup.hasUserReacted}">
-                    <t t-esc="messageReactionGroup.count"/>
+                <span class="o_MessageReactionGroup_count mx-1" t-att-class="{'o-hasUserReacted': messageReactionGroupView and messageReactionGroupView.messageReactionGroup.hasUserReacted}">
+                    <t t-esc="messageReactionGroupView and messageReactionGroupView.messageReactionGroup.count"/>
```

Comment:
Don't need guard `messageReactionGroupView and` (already covered by `t-if="messageReactionGroupView"` above)